src/test/java/racingcar/UnitTest.java

```diff
@@ -0,0 +1,193 @@
+package racingcar;
+
+import camp.nextstep.edu.missionutils.test.NsTest;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static camp.nextstep.edu.missionutils.test.Assertions.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class UnitTest extends NsTest {
+
+    @Test
+    @DisplayName("readInput 테스트")
+    void readInputTest() {
+        String input = "car1,car2,car3\n";
+        System.setIn(new ByteArrayInputStream(input.getBytes()));
+        String result = Application.readInput();
+        assertThat(result).isEqualTo("car1,car2,car3");
+    }
+
+    @Test
+    @DisplayName("splitInput 작동테스트")
+    void splitInputTest() {
+        List<RacingCar> resultCars = Application.splitInput("car1,car2, car3");
+        assertThat(resultCars).extracting("name").containsExactly("car1", "car2", "car3");
+    }
+
+    @Test
+    @DisplayName("splitInput 같은 이름이 있을 경우")
+    void slitInputTest2() {
+        assertThatThrownBy(() -> Application.splitInput("car1,car2,car1"))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("동일한 이름을 가진 차종을 입력하실 수 없습니다.");
+    }
+
+    @Test
+    @DisplayName("splitInput 길이 5이상 자동차 있는 경우")
+    void slitInputTest3() {
+        assertThatThrownBy(() -> Application.splitInput("car1,car2,car333"))
+                .isInstanceOf(IllegalArgumentException.class)
+                .hasMessage("자동차 이름은 4자 이하로 입력해주세요.");
+    }
+
+    @ParameterizedTest
+    @CsvSource({"1,1", "2,2", "3,3", "4,4", "5,5", "10000,10000"})
+    @DisplayName("parseTrial 글자수 정상 작동 테스트")
+    void parseTrialTest(String input, int realTrial) {
+        Integer result = Application.parseTrial(input);
+        assertThat(result).isEqualTo(realTrial)
+                .isPositive();
+    }
```

Comment:
```suggestion
    @ParameterizedTest
    @CsvSource({"1,1", "2,2", "3,3", "4,4", "5,5", "10000,10000"})
    @DisplayName("문자열을 숫자로 올바르게 파싱한다")
    void parseTrialTest(String input, int expected) {
        int actual = Application.parseTrial(input);
        assertThat(actual).isEqualTo(expected);
    }
```

1. 테스트에서는 real보다는 actual, expected라는 표현을 쓰는 편이에요.
2. 테스트 케이스가 너무 많아요. String이 int 타입으로 올바르게 파싱되는지만 테스트하고 있네요. (DisplayName도 좀 모호하네요) -1, 0, 1 정도만 테스트해보면 어떨까요? (경계값 테스트에 대해서 알아봅시다)
3. 동등성 검증을 이미 하고 있어서 isPositive 검증은 필요 없어요